lib/util/git/index.ts

```diff
@@ -878,6 +878,7 @@ export async function prepareCommit({
   files,
   message,
   force = false,
+  recreateMergedPr = false,
```

Comment:
I didn't think that `git` was the problem/blocker here. I don't think that this problem requires any changes to the git layer, unless you can convince me I'm missing something?

Reply:
Yor are right. This is residual from old logic, forgot to delete it.